modules/logging-content-filter-drop-records.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-content-filtering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-content-filter-drop-records_{context}"]
+= Configuring content filters to drop unwanted log records
+
+When the `drop` filter is configured, the log collector evaluates log streams according to the filters before forwarding. The collector drops unwanted log records that match the specified configuration.
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have administrator permissions.
+* You have created a `ClusterLogForwarder` custom resource (CR).
+
+.Procedure
+
+. Add configuration for a filter to the `filters` spec in the `ClusterLogForwarder` CR.
++
+The following example shows how to configure the `ClusterLogForwarder` CR to drop log records based on regular expressions:
++
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+  filters:
+  - name: <filter_name>
+    type: drop # <1>
+    drop: # <2>
+      test: # <3>
+      - field: # <4>
```

Comment:
I would put an actual example of regex here since you mention it above.  I pulled these from the dev docs:
`          - field: .message
            matches: (?i)\berror\b
`
and/or
`          - field: .kubernetes.labels."foo-bar/baz"
            matches: .+
`

Reply:
can you give one for `notMatches` so we're covering both in this case?

Reply:
```          
      drop:
        - test: 
          - field: .kubernetes.pod_name
            notMatches: "my-pod"

```
Also, in addition to multiple `field` within a single `test` that all must evaluate to true, you can also have additional tests that are treated as `OR` checks:   
Example:  We would drop if  test <1> OR <2> evaluates to true, but for <2> both `field` specs must be true for it to be evaluated to true.
```
      drop:
        - test: <1>
          - field: .kubernetes.namespace_name
            matches: "^open"
        - test: <2>
          - field: .log_type
            matches: "application"
          - field: .kubernetes.pod_name
            notMatches: "my-pod"
```